src/utils/renderGasFee.ts

```diff
@@ -35,5 +36,6 @@ export async function renderGasFee(
   if (!(sourceFee && destFee)) {
     return "0";
   }
-  return Big(sourceFee).add(Big(destFee)).toString();
+
+  return new BigNumber(sourceFee).plus(new BigNumber(destFee)).toString();
```

Comment:
can we remove bignumber.js from the dependency as well because we can use https://docs.ethers.org/v5/api/utils/bignumber/#BigNumber--types?

Reply:
done